src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Telemetry/VsTelemetryService.cs

```diff
@@ -65,17 +65,15 @@ protected virtual void PostEventToSession(TelemetryEvent telemetryEvent)
 
         public string HashValue(string value)
         {
-            // Don't hash PII for internal users since we don't need to.
-            if (TelemetryService.DefaultSession.IsUserMicrosoftInternal)
-            {
-                return value;
-            }
+            //// Don't hash PII for internal users since we don't need to.
+            //if (TelemetryService.DefaultSession.IsUserMicrosoftInternal)
+            //{
+            //    return value;
+            //}
```

Comment:
Whats the story here?

Reply:
Great catch, that we was me repro'ing a bug.